tokio/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 1.24.0 (January 5, 2022)
+
+### Fixed
+ - rt: improve native `AtomicU64` support detection ([#5284])
+
+### Added
+ - rt: add configuration option for max number of I/O events polled from the OS
+   per tick ([#5186])
+ - rt: add an environment variable for configuring the default number of worker
+   threads per runtime instance ([#4250])
```

Comment:
We don't usually have newlines within individual entries in the changelog.

Reply:
We probably should if the line gets too long. Markdown formatters will insert new lines.